src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_TERM = 7;
+  const SHORT_TERM = 3;
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  const baseCost = days * PRICE_PER_DAY;
+
+  if (days >= LONG_TERM) {
+    return baseCost - LONG_TERM_DISCOUNT;
+  } else if (days >= SHORT_TERM) {
+    return baseCost - SHORT_TERM_DISCOUNT;
+  }
```

Comment:
![image](https://github.com/mate-academy/js_task-transportation-on-vacation/assets/54860641/cf0b7c57-bafb-4a28-8c08-d570fc2f3c95)

```suggestion
  if (days >= LONG_TERM) {
    return baseCost - LONG_TERM_DISCOUNT;
  } 
  if (days >= SHORT_TERM) {
    return baseCost - SHORT_TERM_DISCOUNT;
  }
```